src/resources/js/commerce/gateway/toggler.js

```diff
@@ -72,6 +72,7 @@ tribe.tickets.commerce.gateway = tribe.tickets.commerce.gateway || {};
 		toggle: 'tribe-tickets__commerce-checkout-gateway-toggle',
 		toggleOpen: 'tribe-tickets__commerce-checkout-gateway-toggle--open',
 		toggleButton: 'tribe-tickets__commerce-checkout-gateway-toggle-button',
+		toggleHidden: 'tribe-common-a11y-hidden',
```

Comment:
Normally, [when we use class names or selectors](https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/js/v2/tickets-block.js#L41) we include the dot `.`. And then we have a prototype function which is `.className()` to get it without the dot.

i.e.: `obj.toggler.classes.toggleHidden.className()`.

I think the best idea would be to have consistency with this and keep doing things like we're doing it on the other libraries. (also the key that it's normally used for the a11y class is `hiddenElement` to make it more abstract and not only related to the toggle.)

Reply:
Nice! Didn't know about that. Will implement it.